templates/html/index.html

```diff
@@ -2,6 +2,9 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
+  {% if params.baseHref %}
+   <base href="{{params.baseHref}}"></base>
```

Comment:
As per [HTML documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base) on the base tag, there shouldn't be a closing tag:

```suggestion
   <base href="{{params.baseHref}}">
```